api/managementpb/management.proto

```diff
@@ -0,0 +1,72 @@
+syntax = "proto3";
+
+package management;
+
+option go_package = "managementpb";
+
+import "github.com/mwitkow/go-proto-validators/validator.proto";
+import "google/api/annotations.proto";
+import "inventory/agents.proto";
+import "inventory/services.proto";
+
+// Add MySQL
+
+message AddMySQLRequest {
+  // Node identifier where this instance runs. Required.
+  string node_id = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Unique across all Services user-defined name. Required.
+  string service_name = 2 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Service Access address (DNS name or IP). Required.
+  string address = 3 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // Service Access port. Required.
+  uint32 port = 4 [
+    (validator.field) = {
+      int_gt: 0
+    }
+  ];
+  // MySQL username for scraping metrics.
+  string username = 5 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // MySQL password for scraping metrics.
+  string password = 6;
+  // MySQL username for getting performance data.
+  string qan_username = 7 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
+  // MySQL password for getting performance data.
+  string qan_password = 8;
+}
```

Comment:
I think there should be a way to add only one Agent, or maybe even zero Agents. That what we had for PMM 1.x:
```
pmm-client add -h
This command is used to add a monitoring service.

Usage:
  pmm-admin add [command]

Available Commands:
  […]
  mysql              Add complete monitoring for MySQL instance (linux and mysql metrics, queries).
  mysql:metrics      Add MySQL instance to metrics monitoring.
  mysql:queries      Add MySQL instance to Query Analytics.
[…]
```

(also note how it uses "monitoring service" term 😐)

So I propose to add two boolean flags for that.

Reply:
> also note how it uses "monitoring service" term

Is it comment from Peter about the same thing?
https://confluence.percona.com/display/PMM/Inventory+API?focusedCommentId=21038419#comment-21038419

Reply:
Yes. See also https://confluence.percona.com/display/PMM/PMM+1.x%3A+Managing+external+exporters+with+pmm-managed+and+pmm-admin and https://confluence.percona.com/display/PMM/External+exporters+improvements.
But in the end, we stick with our current definitions for Service and Agent. So let's say something along the lines:
> This API adds MySQL Service and starts several Agents.